draft-law-moq-carp.md

```diff
@@ -121,6 +121,41 @@ to include a new entry, as defined in Table 1 below:
 
 Every Track entry in a CARP catalog MUST declare a "packaging" type value of "cmaf".
 
+## Timeline description
+This specification extends the METADATA element of the timeline track, defined in WARP
+Section 7, in the following four aspects: 
+
+* Specification of a general scheme for metadata signalled through the METADATA element
+  of the timeline track [Ed.Note: This aspect should also be applied to WARP, thus
+  should be moved to WARP later on.]
+* Definition of a namespace for CARP-specific metadata signalled through the METADATA
+  element of the timeline track
+* Addition of metadata signalling of SAP type
+* Addition of metadata signalling of earliest presentation time
+
+
+### General metadata scheme
+When not empty, the string in the METADATA field MUST contain one or more comma
+separated key=value pairs, formatted as Strings as specified in Section 3.3.3 of
+[RFC9651]. For example, the METADATA field can be "key1=1,key2=3268".
+
+### Namespace for CARP-specific metadata signalled through the METADATA element
+For CARP-specific metadata signalled through the METADATA element, the namespace
+is "ietf:moq:warp:timeline:metadata:carp".
```

Comment:
Where is this namespace added? Is it the prefix to the key? If so, would be good to explicitly state that. 

```
ietf:moq:warp:timeline:metadata:carp:SAP_TYPE
ietf:moq:warp:timeline:metadata:carp:EARLIEST_PTS
```

Also, what is the separator between the prefix and the key name. Is it : or . or something else?

```
ietf:moq:warp:timeline:metadata:carp.SAP_TYPE
ietf:moq:warp:timeline:metadata:carp.EARLIEST_PTS
```